### DISCUSSION TITLE
Battery conservation mode - feature or bug?

### DISCUSSION THREAD

**Discussion by WontFixXx** [BartoszCichecki/LenovoLegionToolkit#950] [Q&A] [answered]
Can you please tell me if the laptop should charge / discharge up to 60% when using the conservation mode?

My Legion Pro 7 16IRX8H keeps battery level.

If it was 30% - remains 30%
If it was 90% - it remains 90%

Is this expected behavior or a bug?

**Comment by BartoszCichecki** [comment, marked as answer]
Conservation mode will charge to 80% max on your laptop. If you turn it above 80% it will do nothing. If you turn it on at ie 90% disconnect the laptop and let it drain to below 80%. Then it should stick to 80% max charge.

If it stops charging at anything less than 80%, head over to Discord linked in Readme and look for EC reset guide.
